src/webob/request.py

```diff
@@ -313,7 +313,7 @@ def body_file_seekable(self):
         "int",
     )
     remote_user = environ_getter("REMOTE_USER", None)
-    remote_host = environ_getter("REMOTE_HOST", None, "4.1.9")
+    remote_host = environ_getter("REMOTE_HOST", None)
```

Comment:
This is already fixed in a different branch. Please remove this change for right now.

Reply:
Done in d0eda05.